frontend/webapp/cypress/e2e/onboarding-flow.cy.ts

```diff
@@ -0,0 +1,36 @@
+describe('On Boarding Flow Tests', () => {
+
+        beforeEach(() => {
+            cy.visit('localhost:3000')
+        });
+
+        it('Source page redirects correctly', () => {
+            cy.url().should('eq', 'http://localhost:3000/choose-sources');
+        })
+
+        it('Run on the onboarding flow', () => {
+            // Select All Apps
+            cy.get('[data-cy="choose-source-coupon"]').should('exist').click()
+            cy.get('[data-cy="choose-source-frontend"]').should('exist').click()
+            cy.get('[data-cy="choose-source-inventory"]').should('exist').click()
+            cy.get('[data-cy="choose-source-membership"]').should('exist').click()
+            cy.get('[data-cy="choose-source-pricing"]').should('exist').click()
+
```

Comment:
this can be a custom command if we using microservices demo for the tests
https://docs.cypress.io/api/cypress-api/custom-commands

Reply:
Lo Hevanti :/